release_notes/49-release-notes.adoc

```diff
@@ -0,0 +1,478 @@
+* `admissionControl.enforcement`: Determines if the admission controller has been configured to enforce policies that have enforcement enabled. This field defaults to `Enabled` for new installations.
+
+For more information, see the following documentation:
+
+* xref:../operating/manage_security_policies/use-admission-controller-enforcement.adoc#use-admission-controller-enforcement[Using admission controller enforcement]
+* xref:../installing/installing_ocp/install-secured-cluster-config-options-ocp.adoc#admission-controller-settings_install-secured-cluster-config-options-ocp[Admission controller settings] (Operator)
+* xref:../installing/installing_ocp/install-secured-cluster-ocp.adoc#secured-cluster-services-config_install-secured-cluster-ocp[Configuration parameters] (Helm)
+* xref:..//cloud_service/installing_cloud_ocp/install-secured-cluster-cloud-ocp.adoc#secured-cluster-services-config_install-secured-cluster-cloud-ocp[Configuration parameters] (Helm for {product-title-managed-short})
+* xref:../cli/command-reference/roxctl-sensor.adoc#roxctl-sensor[roxctl sensor command]
+
+//ROX-29160
+[id="redhat-images-signed-release-key_{context}"]
+=== New default system policy for Red Hat image signing
+
+A new default policy, "Red Hat images must be signed by a Red Hat release key", is now available. This policy ensures that Red Hat images are signed by Red Hat's link:https://access.redhat.com/security/team/key[Release Key 3 product signing key]. In addition to ensuring supply chain provenance, this default policy serves as an example of using the "Image Signature" field and combining it with other criteria.
+
+For more information, see xref:../operating/manage_security_policies/security-policy-reference.adoc#high-sev-security-policies_security-policy-reference[High severity security policies].
+
+//ROX-26282
+[id="automatic-baseline-locking_{context}"]
+=== Ability to automatically lock baselines to improve process execution policy
+
+This new feature is a major improvement to our process execution policy. It directly addresses customer challenges by automating the process of locking baselines, which was previously a time-consuming and manual task for each deployment. This frees up security teams to focus on more critical work.
+Furthermore, this change allows for a more proactive security approach. Instead of waiting for a deployment to exist before setting up an alert, you can now define a policy for a specific scope, such as a namespace. Any new deployment in that scope will automatically be raise alerts, ensuring consistent security from day one.
```

Comment:
Suggestion:
- instead of "from day one" (that's marketing jargon)
- use "across all deployments" or something similar

Also:
change "will automatically be raise alerts"
to "will automatically raise alerts"